package.json

```diff
@@ -24,10 +24,12 @@
     "eject": "react-scripts eject",
     "generate-types": "graphql-codegen --config codegen.yml --require dotenv/config",
     "cypress:open": "cypress open",
+    "cypress:run": "cypress run",
     "sitemap": "node src/utils/generateSitemap.js",
     "stylelint:check": "stylelint-config-prettier-scss-check",
```

Comment:
This script gives me an error. What does this do?

Reply:
Runs cypress in the CLI. Only necessary if we want to run cypress tests in the build pipelines.